### PR TITLE
Update readme llp

### PR DESCRIPTION
**Pull request by polandll**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,95 +2,147 @@
...
 
-```sh
-go build
-```
+There are three methods for using `cql-proxy`:
 
-Run against a [DataStax Astra][astra] cluster:
+- Locally build and run `cql-proxy`
+- Run a docker image that has `cql-proxy` installed
+- Use k8ssandra to run `cql-proxy` in a Kubernetes container
```
Comment (mpenick) [inline README.md:34]: k8ssandra is not needed. `cql-proxy` would be deploy in their k8s cluster as a sidecar in there applications pod.

File: README.md [hunk 2]
```diff
@@ -2,95 +2,147 @@
...
+        datastax/cql-proxy:v0.0.4 \
+        --contact-points <cluster node IPs or DNS names>
+      ```
+  If you wish to have the docker image removed after you are done with it, add `--rm` before the image name `datastax/cql-proxy:v0.0.4 `.
 
-The main use for `cql-proxy` is to enable unsupported CQL drivers and existing applications to work
-well with [DataStax Astra][astra]. Unsupported CQL drivers include legacy [DataStax
-drivers][drivers] and non-Datastax, community maintained CQL drivers (including [gocql],
-[rust-driver], etc.). 
+### Use k8ssandra and Kubernetes
```
[the remark this answers is not in the document]
Comment (mpenick) [reply README.md:79]: I'd prefer having more succinct main README and continue to have the kubernetes configuration in another place.

File: README.md [hunk 3]
```diff
@@ -2,95 +2,147 @@
...
+     ```
 
-To see what options are available the `-h` flag will display a help message listing all flags and their corresponding descriptions
-and environment variables
+6. Check the logs:
+    ```sh
+    kubectl logs <deployment-name>
+    ```
+
+## Configuration
```
Comment (mpenick) [inline README.md:143]: This should be higher up (not below the k8s config, if that's to remain in the main README).

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (polandll) [reply README.md:79]: Pulled out k8ssandra - sorry for the misinterpretation.

File: README.md (hunk 1 above)
Comment (polandll) [reply README.md:34]: Yup - pulled out.

File: README.md (hunk 3 above)
Comment (polandll) [reply README.md]: moved it.

File: README.md [hunk 4]
```diff
@@ -2,118 +2,192 @@
...
 
-Run against a [DataStax Astra][astra] cluster:
+Please give it a try and let us know what you think!
+## When to use `cql-proxy`
+
+The `cql-proxy` sidecar enables unsupported CQL drivers to work with [DataStax Astra][astra]. These drivers include both legacy DataStax [drivers] and community-maintained CQL drivers, such as the [gocql] driver and the [rust-driver].
+
+`cql-proxy` also enables applications that are currently using [Apache Cassandra][cassandra] or [DataStax Enterprise (DSE)][dse] to use Astra without requiring any code changes.  Your application just needs to be configured to use the proxy.
+
+If you're building a new application using DataStax [drivers], `cql-proxy` is not required, as the drivers can communicate directly with Astra. DataStax drivers have excellent support for Astra out-of-the-box, and are well-documented in the [driver-guide] guide.
```
Comment (mpenick) [inline README.md:31]: > If you're building a new application **using** DataStax [drivers]

If you're building a new application **use** (the) DataStax [drivers].

File: README.md [hunk 5]
```diff
@@ -2,118 +2,192 @@
...
 ```
 
-or using Docker as
+To pass configuration to `cql-proxy`, either command-line flags or environment variables can be used. Using the `docker` method as an example, the follwing samples show how username, password and bundle are defined with each method.
+### Using flags
 
 ```sh
 docker run -v <your-secure-connect-bundle.zip>:/tmp/scb.zip -p 9042:9042 \
-  --rm datastax/cql-proxy:v0.0.4 \
+  --rm datastax/cql-proxy:v0.0.2 \
```
Comment (mpenick) [inline README.md:59]: This should be the latest version `0.0.4` instead of `0.0.2`.

**Review by mpenick — APPROVED** [review]

File: README.md (hunk 5 above)
Comment (polandll) [reply README.md:59]: Doing this for a readme in md actually seems to require pre-processing: https://stackoverflow.com/questions/54740161/use-variable-in-markdown-to-represent-url-base

If you were open to changing the readme to asciidoc, then such variable substitutions are easy. I created a readme.adoc here, if you want to see: https://github.com/polandll/antora-testing/tree/test_readme.

**Pull request merged**